packages/editor-ui/src/views/NodeView.vue

```diff
@@ -2033,6 +2039,8 @@ export default mixins(
 				}
 				this.stopLoading();
 			});
+			
+			this.externalHooks().onExternalHookEvent('nodeView.mounted', this.$store.state);
```

Comment:
Providing `store.state` should not be needed.

Reply:
leftover 🤦